index.html

```diff
@@ -540,48 +462,73 @@ <h2>
-      </p>
-      <p>
-        Whenever <a>user agent</a> acquires or releases a wake lock, the
-        <a>user agent</a> MUST perform the following steps for each
-        <a>WakeLock</a> object:
-      </p>
-      <ol>
-        <li data-tests="wakelock-type.https.html">If <a>WakeLock</a> object's
-        <a data-lt="WakeLock.type">type</a> attribute is not equal to
-        <var>type</var>, abort these steps.
-        </li>
-        <li data-tests="wakelock-onactivechange.https.html">
-          <a>Queue a task</a> which updates the <a>WakeLock</a> object's
-          <code>active</code> attribute and <a data-lt="fire an event">fires an
-          event</a> named <code>activechange</code> at the <a>WakeLock</a>
-          object.
-        </li>
-      </ol>
-      <p data-tests=
-      "wakelock-onactivechange.https.html, wakelock-state-is-global.https.html, wakelock-applicability-manual.https.html, wakelock-document-hidden.https.html">
-        In the task described above, the <a>WakeLock</a> objects's
-        <code>active</code> attribute MUST be set to <code>true</code> if the
-        wake lock has been acquired or to <code>false</code> if the wake lock
-        has been released.
-      </p>
+
+      <section> <h3>Acquire wake lock algorithm</h3>
+        To <dfn>acquire a wake lock</dfn> for a given <a>WakeLock</a> object
+        <var>lock</var>, run these steps <a>in parallel</a>:
+        <ol>
+          <li>
+            Let <var>type</var> be <var>lock</var>'s type.
+          </li>
+          <li>
+            If the wake lock for type <var>type</var> is not <a data-lt=
+            "applicable wake lock">applicable</a>, return <code>false</code>.
+          </li>
+          <li>
+            Ask the underlying operation system to
+            <a data-lt="acquire wake lock">acquire the wake lock</a> of
+            type <var>type</var> and let <var>success</var> be the
+            result of the operation.
+          </li>
+          <li data-tests="wakelock-onactivechange.https.html">
+            If <var>success</var> is different from <var>lock</var>'s
+            active, set <var>lock</var>'s active to <var>success</var>
+            and <a data-lt="fire an event">fire an event</a> named
+            <code>"activechange"</code> at <var>lock</var>.
+          </li>
+          <li>
+            Return <var>success</var>.
+          </li>
+        </ol>
+      </section>
+
+      <section> <h3>Release wake lock algorithm</h3>
+        To <dfn>release a wake lock</dfn> for a given <a>WakeLock</a> object
+        <var>lock</var>, run these steps <a>in parallel</a>:
+        <ol>
+          <li>
+            Let <var>type</var> be <var>lock</var>'s type.
+          </li>
+          <li>
+            Ask the underlying operation system to
+            <a data-lt="release wake lock">release the wake lock</a> of
+            type <var>type</var> and let <var>success</var> be the
+            result of the operation.
```

Comment:
Same here, not clear what the result is.

Reply:
fixed